superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/EchartsMixedTimeseries.tsx

```diff
@@ -122,7 +122,6 @@ export default function EchartsMixedTimeseries({
           const filters: QueryObjectFilterClause[] = [];
           filters.push({
             col: formData.granularitySqla,
-            grain: formData.timeGrainSqla,
```

Comment:
@michael-s-molina For some reason, having the grain in the filter object made it return no results.  Removing it fixed it but let me know if you think this will cause other problems.

Reply:
We definitely need the grain because that's how the server-side will interpret the timestamp. So we need to investigate why it's breaking.

Reply:
Reverted these changes in this PR pending solution